lib/elastic_record/persistence_override.rb

```diff
@@ -0,0 +1,7 @@
+module ElasticRecord
+  module PersistenceOverride
+    def save(*)
+      true
+    end
```

Comment:
On closer inspection, it looks like overriding `_create_or_update` is the best choice. Overriding `save` is wrong because it blows away validation logic.

Reply:
I saw that when I was investigating this feature. My concern about it is that it ties into implementation details for ActiveRecord and so we run the risk of being tied to minor Rails versions, like Superstore.

Reply:
Yup, which is ugly.